README.md

```diff
@@ -1,70 +1,62 @@
-# Getting Started with Create React App
+# React-Capstone
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> Add comments and likes to display page
+> Add tests to count for comments, likes and displaying items
+> Create HTML, CSS file
+> Create JavaScript file which manipulates the DOM; adding clicks to elements and adding fucntions to remove and item from the list
```

Comment:
- [ ] Kindly add a description of your application. This description you have here is confusing because I didn't see anywhere in your project that has comments and likes. Please add a description based on this current project.